### ISSUE TITLE
Export a report with Fulcrum-generated handles

### ISSUE THREAD

**Issue by mbakeryo** [mlibrary/heliotrope#269]
**USER STORY**

As a platform admin, I should be able to run a report from command line that will show me all of the handles for each of the assets that I need handles for so I can embed into my ebook.
- [ ] Determine the metadata to use to associate the asset to the handle (file name, title?)

**Comment by mbakeryo** [comment]
Blocked by automated handle service.

**Comment by mbakeryo** [comment]
@jgmorse Do you have any sense of the status of this ticket? It talks specifically about command line, but I wonder if this should be reworked for the UI/dashboard? Also, not sure about the automated handle service aspect.

**Comment by mbakeryo** [comment]
No one has requested this, so we're putting this as low priority.
